notebooks/cohort/case_study/case_1.ipynb

```diff
@@ -0,0 +1,3016 @@
+{
```

Comment:
I have some copyediting changes I'd like to make. I think it will be easier if I make those changes after you check this in, and then I can submit a PR with my suggested changes. (Similarly for the next case studies and other notebooks.) Will that work for you?

Reply:
That works! Or, if you want, you can also make a new commit to this branch before we merge it. Whatever works best for you.